.gitignore

```diff
@@ -17,6 +17,8 @@ dist-ssr
 
 # but allow .env.example
 !.env.example
+!.env.test.baseSepolia
+!.env.test.optimismSepolia
```

Comment:
I don't think it makes sense to have these files included in the repository. It's helpful for testing this PR, but not broadly applicable. Let's remove both of them from the PR and revert this change to `.gitignore`.

Reply:
Done ✅